docs/benchmark/ftlib_benchmark.md

```diff
@@ -118,14 +118,22 @@ model trainable parameters is 23,739,492 (90.56) because there are 100 classes.
 
 | Workers   | speed | speed-up ratio |total task time  | allreduce time| tensor.numpy() time| apply_gradients |
 | --------- | ----- | ------------- |--------------- | -------- | ------ | ---------- |
-| 1 (local) | 168 images/s| 1| 6.1s  |  - | - |  4.16s |
-| 2  | 148 images/s| 0.88 | 13.76s | 10.36s | 5.04s | 1.35s |
-| 4  | 228 images/s| 1.35 | 18s | 14.67s |  5.14s | 1.30s |
+| 1 (local) | 145 images/s| 1| 6.1s  |  - | - |  4.16s |
+| 2  | 148 images/s| 1.02 | 13.76s | 5.32s | 5.04s | 1.35s |
+| 4  | 276 images/s| 1.90 | 18s | 9.53s |  5.14s | 1.30s |
 
 When we run the model locally, CPU timing is not accurate
 because of async behavior of GPU ops.
 
-![Resnet50](https://user-images.githubusercontent.com/18071380/88752262-0fdd2800-d18c-11ea-8b37-83ba5662e53d.png)
+Further, we use Horovod to test the performance with the same dataset and model.
```

Comment:
In addition to FTlib, we test the performance of Horovod with the same training job.